theme/js/components/dataset/search-result.vue

```diff
@@ -0,0 +1,107 @@
+<!-- This is similar to the Jinja2 `dataset/search-result.html` template but in
+Vue. -->
+
+<template>
+  <article class="dataset-card dataset-search-result">
+    <div class="card-logo" v-if="organization">
+      <Placeholder
+        type="dataset"
+        :src="organization.logo_thumbnail"
+        :alt="organization.name"
+      />
+    </div>
+    <div class="card-logo" v-else-if="owner">
+      <Placeholder
+        type="dataset"
+        :src="owner.logo_thumbnail"
+        :alt="owner.name"
+      />
+    </div>
+    <div class="card-logo" v-else>
+      <Placeholder type="dataset" />
+    </div>
+    <div class="card-data">
+      <h4 class="card-title">{{ title }}</h4>
+      <div class="card-description text-grey-300 mt-xs">
+        {{
+          description.length > 300
+            ? description.slice(0, 300) + "…"
```

Comment:
This might be dangerous : what if my space bar malfunctions and I "accidentally" type (or paste) a 500 lines SEO-keywords-stuffed description without a single space (and with weird UTF-8 chars to make them readable enough) ?

Reply:
True. But I'm sure someone on the Internet fixed that problem for you :p Never mind for this PR anyway.

Reply:
I'll put this logic in a filter nonetheless so we can bikeshed this later !

(ps : Just discovered that "Filters are removed from Vue 3.0 and no longer supported. Instead, we recommend replacing them with method calls or computed properties." -> I'm a visionary 😎 )